Update NetworkTraceConfiguration.cs

cristhianu opened the pull request:
Adding checking for when adding App Settings and using default CollectNetworkTrace.exe.config

cristhianu committed:
Update NetworkTraceConfiguration.cs

Adding checking for when adding App Settings and using default CollectNetworkTrace.exe.config

puneet-gupta commented on CollectNetworkTrace/NetworkTraceConfiguration.cs:
```diff
@@ -65,7 +65,7 @@ private static string GetEnvironmentVariable(string name)
         {
             var value = ConfigurationManager.AppSettings[key];
-            if (string.IsNullOrWhiteSpace(value))
+            if (string.IsNullOrWhiteSpace(value) || value.StartsWith("your", StringComparison.OrdinalIgnoreCase))
```
There is a small problem with this change. If the value is the default one in config, then the variable value will get assigned to EMPTY if the APP SETTING does not exist so the check that checks for string `your` just a few lines below will never get called. So do something like this

```
var envVar = GetEnvironmentVariable(key);
if (!string.IsNullOrEmpty(envvar))
{
    value = envvar;
}
```

cristhianu replied:
Done. I fixed it but I used Environment.GetEnvironmentVariable(key) instead as I was getting the "Failed to read environment variable" exception. I tested it and it worked.

cristhianu committed:
Update NetworkTraceConfiguration.cs

Changing to:
var envVar = Environment.GetEnvironmentVariable(key);
            if (!string.IsNullOrEmpty(envVar))
            {
                value = envVar;
            }
Fixing typo in "Please specifiy" to "Please specify"
